bank-sdk/example-app/src/androidTest/java/net/gini/android/bank/sdk/exampleapp/ui/screens/DigitalInvoiceScreen.kt

```diff
@@ -49,7 +47,7 @@ class DigitalInvoiceScreen {
                 .className("android.widget.Button")
                 .text("Get Started")
         )
-        if (onboardingScreenButton.isClickable && onboardingScreenButton.exists()) {
+        if (onboardingScreenButton.isClickable() && onboardingScreenButton.exists()) {
```

Comment:
[Request] Please first check if it exists, and only then if it's clickable 😄 As we discussed, if it does not exist but we check if it's clickable, then it will throw an error. 
Checking if it exists first will avoid this error, and simply not check if it's clickable.

Reply:
The condition would now check for the existence of `onboardingScreenButton ` like this

`if (onboardingScreenButton.waitForExists(5000) && onboardingScreenButton.isClickable) {`